module/mfrc522_spi.c

```diff
@@ -11,3 +11,24 @@ struct address_byte address_byte_build(u8 mode, u8 addr)
 
 	return byte;
 }
+
+u8 mfrc522_register_read(struct spi_device *client, u8 reg)
+{
+	struct address_byte reg_read =
+		address_byte_build(MFRC522_SPI_READ, reg);
+	u8 value;
+
+	spi_write(client, &reg_read, 1);
+	spi_read(client, &value, 1);
```

Comment:
Like below, we could also do a `spi_write_then_read()`
```suggestion
        spi_write_then_read(client, &reg_read, 1, &value, 1);
```